src/wallet/wallet2.cpp

```diff
@@ -4075,6 +4293,15 @@ void wallet2::transfer_selected_rct(std::vector<cryptonote::tx_destination_entry
   ptx.construction_data.unlock_time = unlock_time;
   ptx.construction_data.use_rct = true;
   ptx.construction_data.dests = dsts;
+  ptx.construction_data.dest_subaddr = is_subaddress;
+  ptx.construction_data.subaddr_account = get_current_subaddress_account();
+  // record which subaddress indices are being used as inputs
+  std::set<uint32_t> subaddr_indices;
+  for (size_t idx: selected_transfers)
+    subaddr_indices.insert(m_transfers[idx].m_subaddr_index.minor);
+  ptx.construction_data.subaddr_indices.clear();
```

Comment:
Not in my current design. Funds received by different accounts are completely separate and can never been spent together. Isn't that something we agreed on after the discussion?